### DISCUSSION TITLE
B站取消了返回旧版网页端，求求大佬们做一个能返回旧版网页端的功能！

### DISCUSSION THREAD

**Discussion by SolaWang** [the1812/Bilibili-Evolved#4206] [功能建议 / Ideas]
如题，B站新版网页端属实用的难受，字体和视频页面整的巨大过于晃眼，希望我不是唯一一个觉得新版网页端难看的。

**Comment by the1812** [comment]
建议去这问
https://github.com/MotooriKashin/Bilibili-Old

**Comment by Tinhone** [comment]
自荐脚本 https://greasyfork.org/zh-CN/scripts/455571
